libbeat/publisher/pipeline/pipeline.go

```diff
@@ -93,6 +94,7 @@ type Settings struct {
 type Annotations struct {
 	Beat  common.MapStr
 	Event common.EventMetadata
+	Host  common.MapStr
```

Comment:
Hmmm.... do we consider to introduce more namespaces? If so, how about changing Annotations to:

```
type Annotations struct {
  Event common.EventMetadata // additional event fields configured by user
  Builtin common.MapStr
}
```

And then init Annotations with:

```
Annotations{
  Event: config.EventMetaData,
  Builtin: common.MapStr{
    "beat": ...
    "host": ...
  },
}
```

This way the logic how/when the builtin namespaces are merged needs to be done only once.